arangod/RocksDBEngine/RocksDBTransactionState.h

```diff
@@ -147,6 +147,15 @@ class RocksDBTransactionState final : public TransactionState {
   ///        Used to update the estimate after the trx committed
   void trackIndexRemove(TRI_voc_cid_t cid, TRI_idx_iid_t idxObjectId, uint64_t hash);
 
+  /// @returns tick of last operation in a transaction
+  /// @note not in recovery: the value is valid only after transaction is committed
+  /// @note in recovery: the value is valid immediately after transaction is started
+  TRI_voc_tick_t lastOperationTick() const noexcept final { return _lastOperationTick; }
```

Comment:
seems like this has about the same purpose as `_lastWrittenOperationTick` in mmfiles. Perhaps move this up into the base-class ?

Reply:
@graetzer sure, that is on my list :-)